src/app/partials/nav/header-static/header-static.component.scss

```diff
@@ -3,19 +3,251 @@
 header{
 	position: fixed;
 	left: 0;
-	width: 100%;
 	top: 0;
 	color: white;
 	background-color: rgba(255,255,255,0);
 	width: 100%;
-	padding: 10px 40px;
 	box-sizing: border-box;
 	z-index: 1000;
-	font-size: $fs-14;
 	transition: all 0.2s ease-in-out;
 	.logo{
 		img{
 			width: 30px;
 		}
 	}
 }
+.dark-overlay{
+	width:100%;
+	height:100%;
+	position:fixed;
+	z-index:100;
+	background:rgba(0,0,0,0.7);
```

Comment:
Use variable for bg colors @live-wire

Reply:
Used `variable.scss` for all rgba values